src/templates/doe_workflow_hpwh.mdx

```diff
@@ -2,23 +2,7 @@
 
 <Tabs>
   <Tab eventKey="project" style={{paddingTop:"1rem"}} title="Project">
-    The data for this tab should be know prior to the initial site visit.
```

Comment:
I suggest keeping line 5 "The data for this ..." separate from ProjectInfoInputs. I can imagine using a different sentence here for different workflows. For example, I think the building number photo input is included in the first tab in one of the workflows, and that photo can't be taken prior to the initial site visit.

Also, it should be " ... known ...". The "n" is missing.

Reply:
Moved it accordingly